internal/pkg/remote/config.go

```diff
@@ -87,6 +87,40 @@ func (a *StorageApi) ListComponentsRequest(branchId model.BranchId) *client.Requ
 		})
 }
 
+// ListConfigMetadataRequest https://keboola.docs.apiary.io/#reference/components-and-configurations/copy-configuration-rows/search-component-configurations
+func (a *StorageApi) ListConfigMetadataRequest(branchId model.BranchId) *client.Request {
+	metadata := ConfigMetadataResponse{}
+	return a.
+		NewRequest(resty.MethodGet, "branch/{branchId}/search/component-configurations").
+		SetPathParam("branchId", branchId.String()).
+		SetQueryParam("include", "filteredMetadata").
+		SetResult(metadata)
+}
+
+type (
+	ConfigMetadataResponse []ConfigMetadataWrapper
+	ConfigMetadataWrapper  struct {
+		ComponentId model.ComponentId `json:"idComponent"`
+		ConfigId    model.ConfigId    `json:"configurationId"`
+		Metadata    []ConfigMetadata  `json:"metadata"`
+	}
+	ConfigMetadata struct {
+		Id        string `json:"id"`
+		Key       string `json:"key" diff:"true"`
+		Value     string `json:"value" diff:"true"`
+		Timestamp string `json:"timestamp"`
+	}
+)
+
+func (c ConfigMetadataResponse) GetMetadataMapByConfigKey(branchId model.BranchId) map[model.ConfigKey][]ConfigMetadata {
```

Comment:
`Get` sa nezvykne v Go pisat v nazve metod (jedine ze sa neda inak, a mas field s rovnakym nazvom).
Uplne by tu stacilo aj `MetadataMap`, zvysok je jasny z typu.

To len poznamka bokom, moze to zostat.

Reply:
ok, upravím ještě